scif_app_recipes/ndcctools_v7.1.2_ubuntu16.04.scif

```diff
@@ -15,8 +14,8 @@
         --change-directory="${SCIF_APPDATA}" \
         --makeflow-log="${SCIF_APPDATA}/workflow.jx.makeflowlog" \
         --batch-log="${SCIF_APPDATA}/workflow.jx.batchlog" \
-        "/scif/apps/odm_workflow/src/short_workflow.jx"'
-    [ ! -f "/output/canopycover_fileslist.json" ]; /scif/apps/makeflow/src/prep-canopy-cover.sh "${1}"
+        "/scif/apps/odm_workflow/src/workflow.jx"'
```

Comment:
P.S. Regarding our discussion about how to parameterize the SCIF entrypoint for a Makeflow in order to add the native Makeflow clean ability, but not adding extra entrypoints, nor extra shell logic, I would change this line like so:

```diff
-        "/scif/apps/odm_workflow/src/workflow.jx"'
+        "${3}" "/scif/apps/odm_workflow/src/workflow.jx"'
```

(Note: Replace `${3}` with whatever number designates the last parameter for the entrypoint.)

Then to clean a run of this workflow you would run this:

```bash
docker run --rm <all the usual stuff> agdrone/canopycover-workflow:latest run odm_workflow plot_shapes.shp my_input my_output --clean
```

Reply:
This would also allow additional flags to be added as needed.

Reply:
Yes.

I worry about relying on numbered arguments like this (error-prone and not super clear), so it might be worth looking at using `getopts`:

- <https://unix.stackexchange.com/a/129401>
- <https://wiki.bash-hackers.org/howto/getopts_tutorial>

Reply:
There is a further simplification which I would really like, which is to not have SCIF entrypoints for each different workflow. 

Instead we would expose a single `makeflow` app in the container. Then we can have shell scripts (outside the container) for each different workflow which call the `makeflow` entrypoint with different parameters. 

Another benefit is that a container like this becomes less like an appliance and more like a toolbox. As a result user can create & test their own workflows easily.

Reply:
@julianpistorius If these last two items are not requirements for this PR, can you write issues for them?

Reply:
There is already an issue for clean: https://github.com/AgPipeline/issues-and-projects/issues/165

Reply:
Created issue: https://github.com/AgPipeline/issues-and-projects/issues/232

None of these comments are a requirement for this PR.